source/elements/oneDAL/source/data_management/index.rst

```diff
@@ -244,54 +228,43 @@ in-memory numerical :txtref:`dataset`. It allows:
 
 For details, see :txtref:`accessors` section.
 
-Use-case example for table, accessor and table builder
-------------------------------------------------------
-
-This section provides a basic usage scenario of the :txtref:`table`,
-:txtref:`table-builder`, and :txtref:`accessor` concepts and demonstrates the
-relations between them. :txtref:`The following diagram
-<data_management_sequence_diagram>` shows objects of these concepts, which are
-highlighted by colors:
+Example of interaction between table and accessor objects
+---------------------------------------------------------
 
-- :txtref:`Table builder <table-builder>` objects are blue.
+This section provides a basic usage scenario of the :txtref:`table` and
+:txtref:`accessor` concepts and demonstrates the relations between them.
+:txtref:`The following diagram <table_accessor_usage_example>` shows objects
+of these concepts, which are highlighted by colors:
 
-- :txtref:`Table <table>` objects are cyan.
+- :txtref:`table` object is dark blue.
 
-- :txtref:`Accessors <Accessor>` are yellow.
+- :txtref:`accessor` is orange.
 
-- Grey objects are not a part of |dal_short_name| specification and they are
-  provided just for illustration purposes.
+- :txtref:`table_metadata` is light blue.
 
-.. _data_management_sequence_diagram:
+.. _table_accessor_usage_example:
 
-.. image:: _static/table_builder_accessor_sequence.png
+.. image:: _static/table_accessor_usage_example.png
   :width: 800
   :alt: Sequence diagram of accessor-builder-table relations
 
 To perform computations on a dataset, one shall create a :txtref:`table` object
-first. It can be done using a :txtref:`data-source` or a :txtref:`table-builder`
-object depending on the situation. The diagram briefly shows the situation when
-:txtref:`table` is interactively created from a various external entities (not
-shown on a diagram) using a :txtref:`table-builder`.
+first. It can be done using a :txtref:`data-source` or directly from
+user-defined memory. The diagram briefly shows the situation when some
+:txtref:`table` object `t` is created from the data provided by user (not shown
+on a diagram). While table creation, object of table metadata `tm` is created
```

Comment:
while table creation - wording is unclear

Reply:
rewritten